book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+
+  def add_rental(rental)
+    @rentals << rental
```

Comment:
- You should create the new rental achieving the `has-many` association in order to achieve that you need something like that `Rental.new(date, self, person)` in the add_rental method

Reply:
@youmari thank you for the review, but I was wondering the ` has-many ` association is achieved because we have an array of rentals in the books constructor, and the param `rental` is a rental object, so can you please explain more and ty